format/index.md

```diff
@@ -162,25 +168,23 @@ In addition, a resource object **MAY** contain any of these top-level members:
 
 Here's how an article (i.e. a resource of type "articles") might appear in a document:
 
-```javascript
-// ...
-{
-  "type": "articles",
-  "id": "1",
-  "attributes": {
-    "title": "Rails is Omakase"
-  },
-  "relationships": {
-    "author": {
-      "links": {
-        "self": "/articles/1/relationships/author",
-        "related": "/articles/1/author"
-      },
-      "data": { "type": "people", "id": "9" }
+```json
+  {
```

Comment:
In lieu of `// ...` comments to indicate that this is only part of a larger JSON payload, I've increased the indentation of this section.